### PR TITLE
Always use ISO 8601 as the DateTime format string

### PR DESCRIPTION
**Pull request by ExE-Boss**
Fixes <https://github.com/Ryujinx/Ryujinx/issues/2660>

### CODE REVIEW

File: Ryujinx/Ui/MainWindow.cs [hunk 1]
```diff
@@ -857,7 +857,7 @@ public void LoadApplication(string path, bool startFullscreen = false)
 
                 _applicationLibrary.LoadAndSaveMetaData(_emulationContext.Application.TitleIdText, appMetadata =>
                 {
-                    appMetadata.LastPlayed = DateTime.UtcNow.ToString();
+                    appMetadata.LastPlayed = DateTime.UtcNow.ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss'Z'");
```
Comment (AcK77) [inline Ryujinx/Ui/MainWindow.cs:860]: What about that?
```suggestion
                    appMetadata.LastPlayed = DateTime.UtcNow.ToString("o");
```

Comment (ExE-Boss) [reply Ryujinx/Ui/MainWindow.cs:860]: `ToString("o")` includes microseconds.

Comment (AcK77) [reply Ryujinx/Ui/MainWindow.cs:860]: Right! Then it should be `"yyyy-MM-ddTHH:mm:ssZ"` instead of `"yyyy'-'MM'-'dd'T'HH':'mm':'ss'Z'"`

Comment (ExE-Boss) [reply Ryujinx/Ui/MainWindow.cs:860]: @AcK77
The `'` are necessary to escape the `-`, `:`, `T`, and `Z`, it’s what the [`System.Globalization.CultureInfo.InvariantCulture`](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture)’s [`DateTimeFormat`](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo) is configured as.

Comment (AcK77) [reply Ryujinx/Ui/MainWindow.cs:860]: image
Both produce the same result, but well...

**Comment by TSRBerry** [comment]
Closing this, since #4861 was merged.

**Pull request closed without merging**